contributing.md

```diff
@@ -0,0 +1,17 @@
+# Sanoma Learning Design System contributing guide
+
+We would like to welcome all contributors.
+
+These are the basic contributing rules:
+- New branch should be created from the `main` branch
+- Branch naming: `feature/[#issueNumber]-...-...` or `fix/[#issueNumber]-...-...`, eg `feature/[#21]-button-component`
```

Comment:
Perhaps `feature/21-button-component` is somewhat cleaner? I'm not sure if square brackets or hash tags need to be escaped in the terminal.